src/screens/ProfileScreen/elements/FollowInfoRow.js

```diff
@@ -3,8 +3,8 @@ import {StyleSheet, Text, View} from 'react-native';
 import PropTypes from 'prop-types';
 import GlobalButton from '../../../components/Button/GlobalButton';
 
-import {colors} from '../../../utils/colors';
 import {getSingularOrPluralText} from '../../../utils/string/StringUtils';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to import `COLORS` from the `utils/theme` module instead of importing `colors` from `utils/colors`. This change seems to be part of a larger refactoring effort to standardize color usage across the application. Ensure that the `COLORS` object contains all the necessary color constants and is correctly exported from the `utils/theme` module.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->